CIP-SUBLEDGER/README.md

```diff
@@ -0,0 +1,179 @@
+---
+CIP: 125
+Title: Arrestable native assets
+Status: Proposed
+Category: Ledger
+Authors:
+    - Micah Kendall <micah@butane.dev>
+Implementors: [Micah Kendall, <you?>]
```

Comment:
```suggestion
Implementors: []
```
The implementors are the empty set for now; this set is understood to include any potential `<you?>` and no doubt you will be adding yourself (before or after merge) once you have the proposed Aiken implementation.

Reply:
Thank you.